feconf.py

```diff
@@ -748,6 +748,7 @@ def get_empty_ratings():
 QUESTION_EDITOR_DATA_URL_PREFIX = '/question_editor_handler/data'
 QUESTION_SKILL_LINK_URL_PREFIX = '/manage_question_skill_link'
 QUESTIONS_LIST_URL_PREFIX = '/questions_list_handler'
+QUESTIONS_COUNT_URL_PREFIX = '/questions_count_handler'
```

Comment:
The handler name is questionCount should we remove `s` from the question here?

Reply:
Yeah right. Done.